pkg/service/controller.go

```diff
@@ -270,55 +268,51 @@ func (c *ControllerService) ControllerExpandVolume(ctx context.Context, req *csi
 	newSize := capRange.GetRequiredBytes()
 
 	klog.Infof("Expanding volume %v to %v bytes.", volumeID, newSize)
-	conn, err := c.ovirtClient.GetConnection()
+	disk, err := c.ovirtClient.GetDisk(volumeID, ovirtclient.ContextStrategy(ctx))
 	if err != nil {
-		msg := fmt.Errorf("failed to get ovirt client connection %w", err)
-		klog.Error(msg)
-		return nil, status.Error(codes.Unavailable, msg.Error())
-	}
-
-	diskResp, err := conn.SystemService().DisksService().DiskService(volumeID).Get().Send()
-	if err != nil {
-		msg := fmt.Errorf("failed to find disk %s, error %w", volumeID, err)
+		msg := fmt.Errorf("error while finding disk %s, error %w", volumeID, err)
```

Comment:
This will be a duplicate error message, the GetDisk will already return an error message with the disk ID included.